packages/cloudflare/test/fixtures/dev-runtime/astro.config.ts

```diff
@@ -5,6 +5,49 @@ export default defineConfig({
 	adapter: cloudflare({
 		runtime: {
 			mode: 'local',
+			type: 'pages',
+			bindings: {
+				"KV": {
+					type: "kv",
+					id: "KV",
+				},
+				"KV_PROD": {
+					type: "kv",
+					id: "KV_PROD",
+				},
+				"COOL": {
+					type: "var",
+					"value": "ME"
+				},
+				"D1": {
+					type: "d1",
+					database_id: "D1",
+					preview_database_id: "D1",
+					database_name: "D1"
+				},
+				"D1_PROD": {
+					type: "d1",
+					database_id: "D1_PROD",
+					preview_database_id: "D1_PROD",
+					database_name: "D1_PROD"
+				},
+				"R2": {
+					type: "r2",
+					bucket_name: "R2",
+				},
+				"R2_PROD": {
+					type: "r2",
+					bucket_name: "R2_PROD",
+				},
+				"DO": {
+					type: "durable-object",
+					className: "DO",
+				},
+				"DO_PROD": {
+					type: "durable-object",
+					className: "DO_PROD",
+				}
+			}
```

Comment:
**[DOCS]**
Feedback on those API options. We do have some boundaries, since we want to keep as close as possible to Cloudflare's version, but we do have some flexibility _(I think)_. WDYT?